docs/general-concepts/forms/client-side-validation.md

```diff
@@ -49,6 +49,17 @@ Also if you specify that the field is required by eg:
 ```
 then the javascript will also verify that a value has been entered into the field.
 
+The validation is performed whenever you click on a `submit` button on the form.
+```html
+<button type="button" class="btn btn-primary" onclick="Joomla.submitbutton('myform.submit')">Submit</button>
+```
+and the validation is triggered within the javascript Joomla.submitbutton function. You may also have a cancel button:
+```html
+<button type="button" class="btn btn-primary" onclick="Joomla.submitbutton('myform.cancel')">Cancel</button>
+```
+The same Joomla.submitbutton function is run, but it checks if the second part of the parameter is 'cancel', and if so it doesn't trigger the validation. 

```

Comment:
This need a correction.

`Joomla.submitbutton` check for cancel action in the form attribute `data-cancel-task="foobar.cancel"`, see https://github.com/joomla/joomla-cms/blob/79572828c54272adee9ae0851a88d02b3c4eb0ca/build/media_source/system/js/core.es6.js#L371

Addittionaly the `Joomla.submitbutton` can be run as: `Joomla.submitbutton(task, formSelector, validate)` where `validate` is a boolean value to enable/disable validation.

Reply:
Yes, I see that the toolbar buttons which involve Save have a form-validation attribute which the Cancel button doesn't have, and that results in them calling submitbutton with validate set to true, whereas the cancel button will result in a call with validate set to false.

Thanks for highlighting this. I'll fix this up when I'm next doing some more documentation.